### ISSUE TITLE
Improve Help message `codemon --help`

### ISSUE THREAD

**Issue by ankushbhardwxj** [ankingcodes/codemon#67]
Currently, the help message is very unstructured and includes long descriptions. This issue involves rewriting the help messages to something short and concise while maintaining a balanced structure.
The help message can be seen on typing `codemon` or `codemon --help`.
```
           ---CODEMON---
A CLI tool to ace competitive programming contests

COMMANDS:

codemon - - - - - - - - - - - - - - -  displays this message
codemon --help - - - - - - - - - - - - shows help
codemon init <contestName> - - - - - - initialises a contest
codemon fetch <contestName> - - - - fetches all the sample test cases for the contest name provided
codemon fetch  - - - - - - - - - - - extracts contest name from contest directory name and fetches all sample test cases
codemon init -n <file> - - - - - - - - creates file with given name
codemon listen - - - - - - - - - - - - compiles and gives output
codemon reg - - - - - - - - - - - - -  Register user details
codemon practice <dirName> - - - - - - Starts a practice session
```

**Comment by lucifer0987** [comment]
@ankingcodes Is this issue still open? I would like to work on it.

**Comment by ankushbhardwxj** [comment]
@lucifer0987 - Yeah, go ahead !

**Comment by ankushbhardwxj** [comment]
fixed by #72
